tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/Console.java

```diff
@@ -39,4 +41,37 @@ public String getText() {
         console.show();
         return console.bot().styledText().getText();
     }
+
+    public void closeConsole() {
```

Comment:
We're in the `Console` class so you don't need the `Console` suffix. You can rename the method as "`close`".